crates/utils/src/settings/structs.rs

```diff
@@ -45,8 +45,8 @@ pub struct Settings {
 #[serde(default)]
 pub struct PictrsConfig {
   /// Address where pictrs is available (for image hosting)
-  #[default(Url::parse("http://pictrs:8080").expect("parse pictrs url"))]
-  #[doku(example = "http://pictrs:8080")]
+  #[default(Url::parse("http://localhost:8080").expect("parse pictrs url"))]
+  #[doku(example = "http://localhost:8080")]
```

Comment:
All the Docker setups specifiy the pictrs url explicitly, so we can set this to localhost and avoid adding more fields to config/config.hjson